tools/keytools/sign.c

```diff
@@ -711,6 +715,14 @@ static int make_header_ex(int is_diff, uint8_t *pubkey, uint32_t pubkey_sz,
             goto failure;
         }
         memset(signature, 0, CMD.signature_sz);
+        if (CMD.manual_sign && CMD.policy_sign) {
```

Comment:
Can you please document these new signing tools arguments? Not clear how to use these?